### ISSUE TITLE
Error is ts-toolbelt package 

### ISSUE THREAD

**Issue by Reflex-Gravity** [wbkd/react-flow#891]
After adding `react-flow-renderer` into my app, I'm getting the below `ts-toolbelt` error.

This issue is mentioned in `easy-peasy` package which is using `ts-toolbelt`: https://github.com/ctrlplusb/easy-peasy/issues/622

I used one of the tutorial example code:
```javascript
import React from "react"
import ReactFlow from "react-flow-renderer"

const elements = [
    {
        id: "1",
        type: "input", // input node
        data: { label: "Input Node" },
        position: { x: 250, y: 25 },
    },
    // default node
    {
        id: "2",
        // you can also pass a React component as a label
        data: { label: <div>Default Node</div> },
        position: { x: 100, y: 125 },
    },
    {
        id: "3",
        type: "output", // output node
        data: { label: "Output Node" },
        position: { x: 250, y: 250 },
    },
    // animated edge
    { id: "e1-2", source: "1", target: "2", animated: true },
    { id: "e2-3", source: "2", target: "3" },
]

function ChatFlowConfig() {
    return (
        <div style={{ height: 300 }}>
            <ReactFlow elements={elements} />
        </div>
    )
}

export default ChatFlowConfig


```

The Error:
```
Failed to compile.

/home/tuser/Projects/flowReact/node_modules/easy-peasy/node_modules/ts-toolbelt/out/index.d.ts
TypeScript error in /home/tuser/Projects/flowReact/node_modules/easy-peasy/node_modules/ts-toolbelt/out/index.d.ts(983,46):
Type expected.  TS1110

    981 |     ```
    982 |     */
  > 983 |     export type NumberOf<N extends number> = `${N}`;
        |                                              ^
    984 | }
    985 | declare module "Any/_Internal" {
    986 |     import { NumberOf as NNumberOf } from "Number/NumberOf";
```

Any insight regarding this would be helpful.

**Comment by Reflex-Gravity** [comment]
I got this fixed by upgrading my TypeScript to ^4.1

**Comment by moklick** [comment]
With v9.0.0 we don't use easy-peasy anymore.
